examples/stream-analytics/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "prefix" {
+  description = "The prefix which should be used for all resources in this example"
+}
+
+variable "location" {
+  description = "The Azure Region in which all resources in this example should be created."
+}
```

Comment:
Could we put a logical default location here?

Reply:
I am happy to, but I was following the other examples in this provider which don't seem to set a default location. If you would like me to still set it, I am fine with it.